lib/rules/no-magic-numbers.js

```diff
@@ -34,7 +48,10 @@ module.exports = {
                 ignore: {
                     type: "array",
                     items: {
-                        type: "number"
+                        anyOf: [
+                            { type: "number" },
+                            { type: "string", pattern: "^(?:0|[1-9][0-9]*)n$" }
```

Comment:
The pattern should probably allow `-` sign because the rule treats unary minus as a part of the number, so `"ignore": ["1n"]`  wouldn't allow `f(-1n)`, same as how `"ignore": [1]` doesn't allow `f(-1)`.